extlinks/links/admin.py

```diff
@@ -19,6 +30,13 @@ class LinkSearchTotalAdmin(admin.ModelAdmin):
 
 class LinkEventAdmin(admin.ModelAdmin):
     list_display = ("link", "timestamp", "domain", "username", "change")
+    list_select_related = ["username", "content_type"]
+    readonly_fields = ["url_pattern_display"]
```

Comment:
Let's add `username` to the read-only fields. Alternatively, we could make it an [autocomplete field]( https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.autocomplete_fields). We would just have to add a `search_field` attribute to `organisations.user`